cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -327,6 +327,30 @@ private synchronized void removeOldUserTasks() {
     }
   }
 
+  /**
+   * Allow other subsystems to retrieve {@link UserTaskInfo} using uuid. This method assumes uuid comes from some internal book
+   * keeping state, not an external http request. Thus it does not check request parameters like
+   * {@link UserTaskManager#getUserTaskByUserTaskId} below does.
```

Comment:
What does `{@link UserTaskManager#getUserTaskByUserTaskId} below does.` mean? Do we assume the placement of where each function will reside in the code will not change?

Reply:
I'll remove `below`